changelog.md

```diff
@@ -6,6 +6,20 @@
 
 The Sentry Raven log appender has been updated to version 8.0.x. Users that have enabled the Sentry Raven appender will need to update their configuration according to the [sentry migration guide](https://docs.sentry.io/clients/java/migration/).
 
+### Deprecations
+
+#### /v2/schema route is Deprecated
+
+The /v2/schema route, and JSON Schema definitions, are deprecated in favor of RAML. They will not be kept up-to-date. The endpoint will be disabled in Marathon 1.7.0 (can be re-enabled via an extended grace-period deprecation flag), and then completely removed in Marathon 1.8.0.
+
+#### Marathon Health Checks Deprecation Schedule Update
+
+Marathon Health Checks will continue functioning, but are not actively supported. You should migrate to Mesos Health checks as soon as possible.
+
+If you launch Marathon 1.7.0 and have app definitions with Marathon Health Checks, then Marathon will refuse to start. You will be able to temporarily re-enable them via an extended grace-period flag.
+
+Marathon Health Checks will be completely removed in Marathon 1.8.0, and Marathon will refuse to start if the extended grace-period deprecation flag is specified.
```

Comment:
what is the "extended grace period deprecation flag"?

Reply:
I second that question.

Reply:
:whi